### PR TITLE
Update GuideStar and compensation information

### PR DESCRIPTION
**Pull request by choldgraf**
This updates our GuideStar links to the latest version in our Shared Drive, and also updates the process around using it to be a bit clearer, and to match the CS&S recommendations (specifically, around baseline benefits rates)

### CODE REVIEW

File: hr/compensation.md [hunk 1]
```diff
@@ -12,27 +12,32 @@ We are [working on an update of our compensation policy](https://docs.google.com
...
 
-- The numbers there are for the **top-paid position**  
-  - Reduce accordingly depending on the level we're hiring for. A good rule of thumb for defining new roles is to expect a 10% salary bump between levels, and a 2% salary bump for steps within a level.
-- Keep the number of unique base salaries low, and round-up to standardize across titles if necessary.
+Here are a few tips to understand the GuideStar reports:
+
+- **The numbers there are for the top-paid position in each category.** This means that for earlier levels in a job category, we need to reduce the total compensation accordingly.
+  - Rule of thumb: Expect a 10% salary bump between job levels, and a 2% salary bump for steps within a level.
+- **This is total compensation, not salary.** GuideStar reports a combination of salary and benefits, keep this in mind when defining our salary bands.
+- **Do not share the GuideStar report externally.** It is a service paid-for by CS&S and should not be shared out of the CS&S organization.
```
Comment (sgibson91) [inline hr/compensation.md:40]: Just wanna check that the above links are private given this message? Anyone can read our Team Compass docs, so I'm wondering if all this linking is appropriate for a private resource?

**Review by damianavila — APPROVED** [review]
LGTM.

File: hr/compensation.md (hunk 1 above)
Comment (choldgraf) [inline hr/compensation.md:40]:
```suggestion
- **Do not share the GuideStar report externally.** It is a service paid-for by CS&S and should not be shared out of the CS&S organization. The links above should only be available to those with a `@2i2c.org` email address.
```

**Comment by choldgraf** [comment]
Great - thanks all, I added a clarification about access. Will merge this one in

**Pull request merged**